collector/multiplier.go

```diff
@@ -0,0 +1,72 @@
+package collector
+
+import (
+	"context"
+	"sync"
+
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+type Config struct {
+	Collectors []Interface
+	Logger     micrologger.Logger
+}
+
+type Multiplier struct {
```

Comment:
That may be my poor english, but `Multiplier` doesn't sound right. It looks more like a `Collection` or `Set`.

Reply:
The name is not the best. I agree. Totally open to change that. Fine either way.